wolfssl/wolfcrypt/types.h

```diff
@@ -1088,21 +1089,28 @@ typedef struct w64wrapper {
         WC_HASH_TYPE_BLAKE2B = 14,
         WC_HASH_TYPE_BLAKE2S = 15,
         #define _WC_HASH_TYPE_MAX WC_HASH_TYPE_BLAKE2S
+        #ifdef WOLFSSL_SM3
+            WC_HASH_TYPE_SM3     = 16,
```

Comment:
Can/should we avoid changing these numbers?

Reply:
fixed - restored numbering and added it to the end